src/config.ts

```diff
@@ -12,12 +12,12 @@ export const spikeServiceURL: string = process.env.SPIKE_SERVICE_URL || 'spike-s
 export const kartoffelQuery: string = `/search?domainusers.datasource=${
   process.env.DOMAINUSERS_QUERY || 'nonExternals'
```

Comment:
Change this to the new one. There isn't "domainusers.datasource" anymore in the new Kartoffel API, it changed to "digitalIdentity .source".

Reply:
I wrote a question about this below.

Reply:
Done.